runtime/simx/vortex.cpp

```diff
@@ -415,6 +437,21 @@ class vx_device {
       return 1;
     }
 
+    // HW: virtual mem allocator has the same address range as global_mem. next step is to adjust it
```

Comment:
I think we can let the virtual memory size and physical memory size be the same, but Hanran wants to give more flexibility on virtual memory size. So it would be the future work.
We might change `GLOBAL_MEM_SIZE` to `VIRT_MEM_SIZE` and delete the comment saying next step.